content/blog/hackaton-2022-building-a-pong.md

```diff
@@ -0,0 +1,212 @@
+
+---
+Title: Hackapong fun at the shop!
+Date: 2022-11-04 13::55:00
+Author: Jean-Philippe Lamebrt - alex Hoguin
+Category: WDA
+Tags: hackathon, wda
+Slug: hackaton-2022-building-a-pong
+Status: published
+---
+## Here Comes the Hackapong 
+Our Yearly hackathon is a very important moment, first of all we use it to bond as a team and work as a team to use our tools and see what we could add in the future. We used this week to build on tools we already use or that are coming to WDA soon.
```

Comment:
```suggestion
Our Yearly hackathon is a very important moment, first of all since we use it to bond and work as a team, and also to use our tools and see what we could add in the future. We used this week to build on tools we already use or that are coming to WDA soon.
```